pkg/ims/models/models.go

```diff
@@ -2,18 +2,19 @@ package models
 
 import (
 	"github.com/dgrijalva/jwt-go"
+	"time"
 )
 
-type Users struct {
-	ID          int      `json:"id"`
-	FirstName   string   `json:"first_name"`
-	LastName    string   `json:"last_name"`
-	Email       string   `json:"email"`
-	MobilePhone string   `json:"mobile_phone"`
-	CreatedAt   string   `json:"created_at"`
-	ModifiedAt  string   `json:"modified_at"`
-	Roles       []string `json:"roles"`
-	Verified    bool     `json:"verified"`
+type User struct {
+	ID          int      	`json:"id" db:"id"`
+	FirstName   string   	`json:"first_name" db:"first_name"`
+	LastName    string   	`json:"last_name" db:"last_name"`
+	Email       string   	`json:"email" db:"email"`
+	MobilePhone string   	`json:"mobile_phone" db:"mobile_phone"`
+	CreatedAt   time.Time   `json:"created_at" db:"created_at"`
+	ModifiedAt  time.Time   `json:"modified_at" db:"modified_at"`
```

Comment:
make an issue to make this field pointer.

Reply:
Issue created: https://github.com/omc-college/management-system/issues/110